akka-actor-typed/src/main/resources/META-INF/native-image/com.typesafe.akka/akka-actor-typed/reflect-config.json

```diff
@@ -9,7 +9,7 @@
     "name" : "MODULE$"
   } ]
 }, {
-  "name" : "akka.actor.typed.internal.jfr.JFRActorFlightRecorder",
+  "name" : "akka.actor.typed.internal.ActorFlightRecorder$",
```

Comment:
This will block not be needed anymore since it isn't an extension, never touched through reflection.